platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/module/http/HttpRequestImpl.java

```diff
@@ -105,9 +105,14 @@ public static void enableLog(boolean enabled) {
     HttpLogger.logEnabled = enabled;
   }
 
+  @Deprecated
   public static void setOkHttpClient(@Nullable OkHttpClient okHttpClient) {
```

Comment:
Would it be a breaking change to take a `Call.Factory` here?

Reply:
Binary incompatible, but source compatible

Reply:
If it's source compatible I'd say go ahead and just change the signature, if you don't mind. 🙂